### ISSUE TITLE
typings.d.ts needs to be updated

### ISSUE THREAD

**Issue by Saikedo** [relative-ci/bundle-stats#1893]
I am encountering the following error when trying to use the plugin.

```
Type 'BundleStatsWebpackPlugin' is not assignable to type '((this: Compiler, compiler: Compiler) => void) | WebpackPluginInstance'.
  Property 'apply' is missing in type 'BundleStatsWebpackPlugin' but required in type 'WebpackPluginInstance'.
```
And indeed, if you check the typings.d.ts file, you can see that BundleStatsWebpackPlugin is declared like this

```
export class BundleStatsWebpackPlugin {
  constructor(options?: Partial<BundleStatsWebpackPlugin.Options>) {}
};
```

instead of this

```
export class BundleStatsWebpackPlugin {
  constructor(options?: Partial<BundleStatsWebpackPlugin.Options>) {}

  apply(compiler: Compiler): void;
};
```

**Comment by vio** [comment]
Hi @Saikedo, thanks for reporting the issue! Are you interested in adding a PR?

**Comment by Saikedo** [comment]
Hi @vio , done. Opened the following PR

https://github.com/relative-ci/bundle-stats/pull/1958

**Comment by vio** [comment]
Thank you @Saikedo! Released as part of [v3.2.4](https://github.com/relative-ci/bundle-stats/releases/tag/v3.2.4)
